db/migrate/20171011150408_remove_fulltext_index_repositories_on_name_index.rb

```diff
@@ -0,0 +1,10 @@
+class RemoveFulltextIndexRepositoriesOnNameIndex < ActiveRecord::Migration
```

Comment:
Are we removing the fulltext support? Isn't it break the search feature? Or is it gonna make it only slower?

Reply:
Fulltext support has been disabled since the very beginning 😃 We should find a proper solution to this (I just opened #1459)

Reply:
I thought that it was enabled since I saw some `search` methods being called around.

Reply:
See a `TODO` I removed on `app/models/repository.rb` 😉

Reply:
👍